src/generation.rs

```diff
@@ -67,6 +68,7 @@ impl Template {
         path: Option<PathBuf>,
     ) -> Result<String> {
         let mut tera = Tera::default();
+        register_all(&mut tera);
```

Comment:
Better to use `tera_text_filters::register_all(&mut tera)` and remove the import so it's clearer what happens

Reply:
Ya I like this idea.

And I might take a look at depending on heck like you suggest.